schemas/io.snowplow.ecomm/click_item/jsonschema/1-0-0

```diff
@@ -0,0 +1,38 @@
+{
+    "description": "Event occurrs when an item (a product or bundle) is clicked on.",
+    "properties": {
+        "item_id": {
+            "type": "string",
+            "description": "ID reference to the item (as listed in item) that was clicked on.",
+            "maxLength": 255
+        },
+        "item_type": {
+            "type": "string",
+            "enum": ["product", "bundle"],
+            "description": "The general type of the item, currently supports the two types of a singular product and a bundle of products.",
+            "maxLength": 255
+        },
+        "click_position": {
```

Comment:
I would be tempted to name it `position` to make it more simlar to other ecommerce implementations. That way feels more familiar.